WhisperAPI/WhisperAPI.Tests/Unit/VersionControllerTest.cs

```diff
@@ -20,7 +20,7 @@ public void When_getting_version_then_return_version()
 
             var apiVersion = result.As<OkObjectResult>().Value as ApiVersion;
 
-            apiVersion.Version.Should().BeEquivalentTo(new ApiVersion().Version);
+            apiVersion.Version.Should().BeEquivalentTo("12");
```

Comment:
Would let it like it was before. Now we would have to change it at 2 places instead of 1